### PR TITLE
Small improvements to plotting vignette

### PR DESCRIPTION
**Pull request by Felixmil**

### CODE REVIEW

**Comment by PavelBal** [comment]
Resolve conflicts and then merge.

File: docs/dev/articles/esqlabsR-plot-results_files/figure-html/plot-time-profile-1.png
Comment (PavelBal) [inline docs/dev/articles/esqlabsR-plot-results_files/figure-html/plot-time-profile-1.png:1]: @Felixmil Axis labels are now always wrapped and look very strange... Can this be fixed?

**Review by PavelBal — CHANGES_REQUESTED** [review]

**Comment by PavelBal** [comment]
Where are the conflicts coming from? Can you resolve?

**Comment by Felixmil** [comment]
A vignette was changed by `{styler}`. Conflict resolved.

**Review by PavelBal — APPROVED** [review]

**Pull request merged**